### PR TITLE
Added values-ko

### PR DESCRIPTION
**Pull request by dalinaum**
Sorry for late PR.

### CODE REVIEW

File: values-ko/strings.xml [hunk 1]
```diff
@@ -2,115 +2,115 @@
...
+    <string name="how_to_play_144_hint"><b>힌트:</b> 2 &#215; 2 &#215; 2 &#215; 2 &#215; 3 &#215; 3 = 144</string>
+    <string name="tap_to_play_random_move">무작위 움직임을 플레이하기 위해 격자를 누르세요!</string>
 
     <!-- SHARING STRINGS -->
     <!-- The title of the activity chooser when you click the "Share" preference in the Settings -->
-    <string name="share_with"></string>
+    <string name="share_with">공유하기</string>
     <!-- The default share message that is used when you click "Share" in the Settings and share with Google+, Facebook, etc. -->
-    <string name="share_app_message"></string>
+    <string name="share_app_message"플레이 스토어에서 #2048++ 앱을 확인하세요!\n\nhttps://play.google.com/store/apps/details?id=com.alexjlockwood.twentyfortyeight</string>
```
Comment (alexjlockwood) [inline values-ko/strings.xml]: Looks like there is a missing ">" here. Would you mind fixing before I merge?

**Comment by alexjlockwood** [comment]
Thanks a lot for the translation! It looks like there is one typo that is causing compilation errors. Would you mind fixing before I merge? Thanks again!

File: values-ko/strings.xml [hunk 2]
```diff
@@ -2,115 +2,115 @@
...
-    <string name="how_to_play_144_paragraph1"></string>
-    <string name="how_to_play_144_paragraph2"></string>
-    <string name="how_to_play_144_hint"></string>
-    <string name="tap_to_play_random_move"></string>
+    <string name="how_to_play_2048_paragraph1">타일을 격자에 옮기기 위해 상하좌우로 미세요. 정확히 같은 숫자를 가진 <b>2개의 타일</b>이 만나면 하나로 합쳐집니다.</string>
+    <string name="how_to_play_2048_paragraph2">각 움직임이 끝나면 <b>하나의 새로운 타일</b>이 무작위로 격자에 배치됩니다. 게임은 더 이상 타일을 옮길 수 없을 때 끝납니다. 2048 타일을 만들 수 있을까요?</string>
+    <string name="how_to_play_243_paragraph1">S타일을 격자에 옮기기 위해 상하좌우로 미세요. 정확히 같은 숫자를 가진 <b>3개의 타일</b>이 만나면 하나로 합쳐집니다.</string>
+    <string name="how_to_play_243_paragraph2">각 움직임이 끝나면 <b>하나의 새로운 타일</b>이 무작위로 격자에 배치됩니다. 게임은 더 이상 타일을 옮길 수 없을 때 끝납니다. 243 타일을 만들 수 있을까요?</string>
+    <string name="how_to_play_16384_paragraph1">타일을 격자에 옮기기 위해 상하좌우로 미세요. 정확히 같은 숫자를 가진 <b>2개 혹은 4개 타일</b>이 만나면 하나로 합쳐집니다.</string>
+    <string name="how_to_play_16384_paragraph2">각 움직임이 끝나면 <b>두개의 새로운 타일</b>이 무작위로 격자에 배치됩니다. 게임은 더 이상 타일을 옮길 수 없을 때 끝납니다. 16384 타일을 만들 수 있을까요?/string>
```
Comment (alexjlockwood) [inline values-ko/strings.xml]: Oops, seems like there is a typo here as well (it should end with "</string>").

Comment (dalinaum) [inline values-ko/strings.xml]: Sorry. I had some mistakes. I have to fix them.

**Comment by alexjlockwood** [comment]
Thanks for the update! I'll test it out again tonight when I get home from work. :)

**Pull request merged**